fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/AnnouncementCard.kt

```diff
@@ -0,0 +1,127 @@
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.StateColor
+import com.microsoft.fluentui.theme.token.controlTokens.*
+
+@Composable
+fun AnnouncementCard(
+    title: String,
+    description: String,
+    buttonText: String,
+    buttonOnClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    previewImageVector: ImageVector? = null,
+    @DrawableRes previewImageDrawable: Int? = null,
+    announcementCardTokens: AnnouncementCardTokens? = null
+) {
+    val token = announcementCardTokens
+        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.AnnouncementCard] as AnnouncementCardTokens
+    val announcementCardInfo = AnnouncementCardInfo()
+    val textColor = token.textColor(announcementCardInfo = announcementCardInfo)
+    val titleColor = token.titleColor(announcementCardInfo = announcementCardInfo)
+    val descriptionTypography =
+        token.descriptionTypography(announcementCardInfo = announcementCardInfo)
+    val titleTypography = token.titleTypography(announcementCardInfo = announcementCardInfo)
+    val previewTitlePadding = token.previewTitlePadding(announcementCardInfo = announcementCardInfo)
+    val titleTextPadding = token.titleTextPadding(announcementCardInfo = announcementCardInfo)
+    val textButtonPadding = token.textButtonPadding(announcementCardInfo = announcementCardInfo)
+    val textHorizontalPadding =
+        token.textHorizontalPadding(announcementCardInfo = announcementCardInfo)
+    val previewPadding = token.previewPadding(announcementCardInfo = announcementCardInfo)
+    val previewCornerRadius = token.previewCornerRadius(announcementCardInfo = announcementCardInfo)
+
+    class CustomBasicCardTokens : BasicCardTokens() {
+        @Composable
+        override fun cornerRadius(announcementCardInfo: BasicCardControlInfo): Dp {
+            return token.cornerRadius(announcementCardInfo = announcementCardInfo)
+        }
+
+        @Composable
+        override fun elevation(announcementCardInfo: BasicCardControlInfo): Dp {
+            return token.elevation(announcementCardInfo = announcementCardInfo)
+        }
+    }
+
+    class CustomButtonTokens : ButtonTokens() {
+        @Composable
+        override fun textColor(buttonInfo: ButtonInfo): StateColor {
+            return token.buttonTextColor(announcementCardInfo = announcementCardInfo)
+        }
+    }
+    BasicCard(
+        modifier = modifier,
+        basicCardTokens = CustomBasicCardTokens() as BasicCardTokens
+    ) {
+        Box(
+            modifier = Modifier
+                .padding(all = previewPadding)
+                .semantics(mergeDescendants = true) {},
+            contentAlignment = Alignment.Center
+        ) {
+            Column(
```

Comment:
merge Box & Column as the Box has only one element

Reply:
Use one column and arrange image, text, button by applying their individual padding token & spacing tokens in between.